### PR TITLE
i18n(pt-BR): Add `astro-components.md` & `astro-pages.md` translations

### PR DESCRIPTION
**Pull request by Yan-Thomas**
Translations for both `astro-components.md` and `astro-pages.md`.

### CODE REVIEW

File: src/pages/pt-BR/core-concepts/astro-components.md [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+const nome = "Astro";
+---
+<div>
+  <h1>Olá {nome}!</h1>
+</div>
+```
+
+#### Atributos Dinâmicos
+
+Estas variáveis locais podem ser utilizadas entre chaves para passar valores à atributos de elementos HTML e componentes:
```
Comment (victor0x16) [inline src/pages/pt-BR/core-concepts/astro-components.md:125]: I don't think there is a `crase` here
```suggestion
Estas variáveis locais podem ser utilizadas entre chaves para passar valores aos/a atributos de elementos HTML e componentes:
```

File: src/pages/pt-BR/core-concepts/astro-components.md [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+// Importação ESM 
+<script>
+  import './algum-script-externo.js';
+</script>
+```
+
+
+## Próximos Passos
+
+📚 Leia sobre os [componentes integrados ao Astro ](https://docs.astro.build/pt-BR/reference/api-reference/#built-in-components).
```
Comment (victor0x16) [inline src/pages/pt-BR/core-concepts/astro-components.md:386]:
```suggestion
📚 Leia sobre os [componentes internos de Astro ](https://docs.astro.build/pt-BR/reference/api-reference/#built-in-components).
```

File: src/pages/pt-BR/core-concepts/astro-pages.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+export async function get({ params, request }: APIContext) {
+  return {
+    body: JSON.stringify({
+      path: new URL(request.url).pathname
+    })
+  };
+}
+```
+
+Como alternativa, você também pode tipar as funções das suas rotas de API com o tipo `APIRoute`. Isso resultará em melhores mensagens de erro quando a sua rota de API retornar o tipo errado:
```
Comment (victor0x16) [inline src/pages/pt-BR/core-concepts/astro-pages.md:111]:
```suggestion
Como alternativa, você também pode definir tipos para as funções das suas rotas de API com o tipo `APIRoute`. Isso resultará em melhores mensagens de erro quando a sua rota de API retornar o tipo errado:
```

File: src/pages/pt-BR/core-concepts/astro-components.md [hunk 4]
```diff
@@ -0,0 +1,388 @@
...
+
+A sintaxe de um componente Astro é um superset de HTML. A sintaxe foi [projetada para parecer familiar a qualquer um com experiência em escrever HTML ou JSX](/pt-BR/comparing-astro-vs-other-tools/#astro-vs-jsx) e adiciona suporte para a inclusão de componentes e expressões do JavaScript. Você pode localizar um componente Astro por sua extensão de arquivo: `.astro`. 
+
+Componentes Astro são extremamente flexíveis. Geralmente, um componente Astro irá conter alguma **UI reutilizável na página**, como um cabeçalho ou um cartão de perfil. Outras vezes, um componente Astro pode conter um pequeno pedaço de HTML, como uma coleção de tags `<meta>` comuns que facilitam trabalhar com SEO. Componentes Astro também pode conter o layout inteiro de uma página.
+
+A coisa mais importante de entender sobre componentes Astro é que eles vão **renderizar HTML durante sua construção**. Mesmo que você rode código JavaScript dentro dos seus componentes, tudo será rodado previamente e então removido da página final que você envia aos seus usuários. O resultado é um site mais rápido, com nenhum JavaScript por padrão.
```
Comment (victor0x16) [inline src/pages/pt-BR/core-concepts/astro-components.md:14]: Why is `rodado` better?
```suggestion
A coisa mais importante de entender sobre componentes Astro é que eles vão **renderizar HTML durante sua construção**. Mesmo que você excute código JavaScript dentro dos seus componentes, tudo será executado previamente e então removido da página final que você envia aos seus usuários. O resultado é um site mais rápido, com nenhum JavaScript por padrão.
```

**Comment by victor0x16** [comment]
@Yan-Thomas, those are just suggestions for improving.

Great work!

File: src/pages/pt-BR/core-concepts/astro-components.md (hunk 1 above)
Comment (Yan-Thomas) [reply src/pages/pt-BR/core-concepts/astro-components.md:125]:
```suggestion
Estas variáveis locais podem ser utilizadas entre chaves para passar valores a atributos de elementos HTML e componentes:
```

File: src/pages/pt-BR/core-concepts/astro-components.md (hunk 2 above)
Comment (Yan-Thomas) [reply src/pages/pt-BR/core-concepts/astro-components.md:386]: Found an extra space in the link

```suggestion
📚 Leia sobre os [componentes internos de Astro](https://docs.astro.build/pt-BR/reference/api-reference/#built-in-components).
```

**Comment by Yan-Thomas** [comment]
Thanks for the feedback @victor0x16!

Comment 'LGTM' when you think we are ready to merge 🚀

**Comment by victor0x16** [comment]
LGTM

**Pull request merged**